### DISCUSSION TITLE
Overall Great Experience, would love few improvements

### DISCUSSION THREAD

**Discussion by alekssze** [webtorrent/webtorrent-desktop#2194] [General]
Wow This is so Easy to Use and The clean UI is great. Works Great.

The things I think need Improvement is, as built in player doesn't support mkv video files it would be nice if the app would open vlc in front of the app Not Behind The App like it is now - https://imgur.com/a/TnsDj0u

**Comment by lukejmorrison** [comment]
Would be great if it had a webinterface so it could be integrated with Servarr etc :)
